simulated_input/Cargo.toml

```diff
@@ -27,4 +27,6 @@ kanata = { path = ".." , default_features = false }
 [features]
 default = ["simulated_output", "tcp_server"]
 simulated_output = ["kanata/simulated_output"]
+simulated_input = ["kanata/simulated_input"]
```

Comment:
This looks unusual. Is this needed to resolve features properly without breaking builds?

Reply:
That's mostly to "sync" with the main crate since these features are interdependent, though technically as far as I understand, they're unique since it's a different package
Not sure if it's needed to resolve features